cloud-provider-aws.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: cloud-provider-aws
+  version: 1.31.1
+  epoch: 0
+  description: The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+    GO111MODULE: "on"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/cloud-provider-aws
+      tag: v${{package.version}}
+      expected-commit: 94cb7eefd72f6577a0f7840f61a47ebd4aed06f4
+
+subpackages:
+  - name: aws-cloud-controller-manager
+    description: The AWS Cloud Controller Manager is the controller that is primarily responsible for creating and updating AWS loadbalancers (classic and NLB) and node lifecycle management.
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/aws-cloud-controller-manager
+          output: aws-cloud-controller-manager
+          ldflags: |
+            -w -s
```

Comment:
-s is something we don't use and -w is handled by go/build pipeline. We can omit `-s -w` from ldflags.

Reply:
Do we add `uses: strip`?

Reply:
-w does the stripping for us so explicitly using `-w` is not required anytime we are using go/build pipeline.